components/site/src/lib.rs

```diff
@@ -1127,6 +1128,18 @@ impl Site {
             })
             .collect::<Result<()>>()
     }
+	
+	/*
+		when running serve we do not want png optimizations, because that's too slow.
```

Comment:
This should only ever ran once for a file, after that it should be stored in `static` like the rest of the image processing so it's ok to run it on `serve`